projects/packages/forms/src/blocks/contact-form/variations.js

```diff
@@ -62,15 +62,16 @@ const variations = compact( [
 	},
 	! isSimpleSite() && {
 		name: 'newsletter-form',
-		title: __( 'Sign-up', 'jetpack-forms' ),
+		title: __( 'Lead capture', 'jetpack-forms' ),
 		description: __(
-			'A simple way to collect information from folks visiting your site.',
+			'A simple way to collect information from folks visiting your site using CreativeMail plugin.',
 			'jetpack-forms'
 		),
 		keywords: [
 			__( 'subscribe', 'jetpack-forms' ),
 			__( 'email', 'jetpack-forms' ),
 			__( 'signup', 'jetpack-forms' ),
+			__( 'lead capture', 'jetpack-forms' ),
```

Comment:
@jeherve did we stop using "block search keyword" translation context for keywords?

Reply:
Done! :-) https://github.com/Automattic/jetpack/pull/32481/commits/a04f22a416f4fb2ab35e4e4d2845c62e25aefbac